submissions/description/unvalidated_redirects_and_forwards/open_redirect/post_based/template.md

```diff
@@ -0,0 +1,23 @@
+# POST-Based Open Redirect
+
+## Overview of the Vulnerability
+
+Open redirects enable an attacker to manipulate a user by redirecting them to a malicious site. A POST-based open redirection was identified which can impact users’ ability to trust legitimate web pages. An attacker can send a phishing email that contains a link with a legitimate business name in the URL and the user will be redirected from the legitimate web server to any external domain. Users are less likely to notice subsequent redirects to different domains when an authentic URL with a valid SSL certificate can be used within the phishing link.
+
+This type of attack is also a precursor for more serious vulnerabilities such as Cross-Site Scripting (XSS), Server-Side Request Forgery (SSRF), Cross-Site Request Forgery (CSRF), or successful phishing attempts where an attacker can harvest users’ credentials or gain users’ OAuth access.
+
+## Business Impact
+
+POST-based open redirects can result in reputational damage for the business as customers’ trust is negatively impacted by an attacker sending them to a phishing site to extract login credentials, or coercing them to send a financial transaction.
+
+## Steps to Reproduce
+
+1. Using a browser, navigate to: {{URL}}
+1. Copy and modify the URI so that the URL redirects to <http://www.bugcrowd.com/>
```

Comment:
A POST based Open Redirect would not have a user modify the link, it would use a HTML page with a burp request saved from the CSRF option. Something more fitting would be
> Open the attached HTML file in the target browser and follow the prompts

Reply:
Thank you mate